docs/sources/shared/reference/components/otelcol-debug-metrics-block.md

```diff
@@ -11,7 +11,14 @@ The following arguments are supported:
 Name                               | Type      | Description                                          | Default | Required
 -----------------------------------|-----------|------------------------------------------------------|---------|---------
 `disable_high_cardinality_metrics` | `boolean` | Whether to disable certain high cardinality metrics. | `true`  | no
+`level` | `string` |  Controls the level of detail for metrics emitted by the wrapped collector. Possible values are `none`, `basic`, `normal` and `detailed` | `detailed`  | no
```

Comment:
> Possible values are `none`, `basic`, `normal` and `detailed`

Normally we'd mention this outside of the table, so that the table size doesn't bubble up :)

Reply:
I moved the enumeration out of the table in HEAD commit :D

Reply:
Ah... I should have caught that :-P